### PR TITLE
Feature/nonhttp

### PR DESCRIPTION
**Pull request by k8soneill**
Rebased for 3.11 and tested with a 3.11 deployment

### CODE REVIEW

**Review by MudasarUKCloud — APPROVED** [review]
Approving this PR.

File: environment_example.yaml [hunk 1]
```diff
@@ -58,7 +58,14 @@ parameter_defaults:
   #  dns: [ "8.8.4.4" ]
   #  gateway: "10.3.1.102"
   #  bastion_ip: "10.3.1.101"
+  #  service_subnet: This should be a /29 starting at .240 unless more addresses are required. .249-.254 will be left for routers "10.3.1.240/29"
   # Example of passing in configuration for SSO
```
Comment (benbacon) [inline environment_example.yaml]: Can you move this to line 68?

**Review by benbacon — CHANGES_REQUESTED** [review]
Minor change required, otherwise looks good

**Comment by benbacon** [comment]
Looks good

**Review by benbacon — APPROVED** [review]

**Review by stevemul — APPROVED** [review]
lgtm

**Pull request merged**